web/src/main/java/com/navercorp/pinpoint/web/service/SpanServiceImpl.java

```diff
@@ -355,6 +355,11 @@ private void transitionException(List<SpanAlign> spanAlignList) {
 
     private StringMetaDataBo selectStringMetaData(String agentId, int cacheId, long agentStartTime) {
         final List<StringMetaDataBo> metaDataList = stringMetaDataDao.getStringMetaData(agentId, agentStartTime, cacheId);
+		if (cacheId == 404) {
```

Comment:
The cacheId is dynamically generated. Developers should not assign the cacheId arbitrarily.